src/lowerCaseEmails.js

```diff
@@ -0,0 +1,93 @@
+const dotenv = require("dotenv");
+
+dotenv.config({path: "../.env" ,silent: true});
+
+const mongoose = require("mongoose");
+
+const {MONGO_URI} = process.env;
+
+const options = {
+  useNewUrlParser: true,
+  useCreateIndex: true,
+  dbName: "shellhacks",
+  useFindAndModify: false
+};
+
+const db = () => Promise.resolve(mongoose.connect(MONGO_URI, options));
+
+db()
+  .then(() => console.log("> 🍃  Mongo connected"))
+  .catch(e => console.log("> Mongo error ", e.message));
+
+  const applicantSchema = new mongoose.Schema({
```

Comment:
you shouldn't be making a new mongoose schema here, just import the already created one in the models folder

Reply:
I tried that and got a syntax error on import.